docs/3.0/field-options.md

```diff
@@ -327,3 +327,48 @@ Now, all fields will have the stacked layout throughout your app.
 :::option `use_resource`
 <!-- TODO: this -->
 :::
+
+:::option `components`
+The field's `components` option allows you to customize the view components used for rendering the field in all, `index`, `show` and `edit` views. This provides you with a high degree of flexibility.
+
+### Ejecting the field components
+To start customizing the field components, you can eject one or multiple field components using the `avo:eject` command. Ejecting a field component generates the necessary files for customization. Here's how you can use the `avo:eject` command:
+
+#### Ejecting All Components for a Field
+`$ rails g avo:eject --field-components <field_type>`
+
+Replace `<field_type>` with the desired field type. For instance, to eject components for a Trix field, use:
+
+`$ rails g avo:eject --field-components trix`
+
+This command will generate the files for both the edit and show components of the Trix field, for each field type the amount of components may vary.
```

Comment:
Let's provide an example where we eject the `index` view too. I immediately asked myself "why don't they eject the index view as well"?
It took a minute to click for me that trix does not have an Index field.